languages/README.md

```diff
@@ -0,0 +1,165 @@
+
+## How to add a new language 
+
+This section is mainly for anyone who needs to build a new language structure.
+
+Fork [cloudevents/spec](https://github.com/cloudevents/spec/tree/main)
+to your Github repo. 
+
+Then check the [language list](languages.md) to see if the language
+you'd like to translate into already exists. If it doesn't exist, you can start to add a new one.
+
+### Step 1. build the structure
+
+Create a directory under `languages` directory, named after the specific 
+[language code](http://www.lingoes.net/en/translator/langcode.htm).
+
+Add a new item to the [language list](languages.md), which linking to the directory 
+you just created. 
+
+### Step 2. copy documents to right location
+
+Copy all original documents except `languages` directory to `languages/{language-code}`.
+There is no need to copy project-related files(i.e.,LICENSE, OWNERS).
+
+For example:
+```
+copy cloudevents/spec.md to languages/zh-CN/cloudevents/spec.md
+copy README.md to languages/zh-CN/README.md
+```
+
+Clear content of copied documents to indicate that the document is ready to be translated.
+
+Then copy the [reviewer list](zh-CN/reviewers.md) and the [translation list](zh-CN/translations.md)
+to `languages/{language-code}`.
+
+Check example structure [here](zh-CN).
+
+## How to publish translation tasks
+
+### Step 1. open a single issue referring to an untranslated document
+
+Create an issue with label "{language-code}-translation" to publish a translation assignment.
+
+Examples:
+```
+Issue tile:        docs: translate cloudevents/spec.md to zh-CN
+Issue content:   
+                   original document: 
+                   https://github.com/cloudevents/spec/blob/main/README.md
+                   target document: 
+                   https://github.com/cloudevents/spec/blob/main/languages/zh-CN/README.md
+```
+
+A reviewer can assign the issue to anyone who comments to do the translation.
+
+### Step 2. open an overall issue to list and track all tasks
+
+Examples:
+```
+Issue tile:         docs: translation list to zh-CN
+Issue content:   
+                    cloudevents: 
+                    Issue url: docs: translate cloudevents/spec.md to zh-CN
+                    Issue url: docs: translate cloudevents/SDK.md to zh-CN
+                    Issue url: docs: translate cloudevents/primer.md to zh-CN
+     
+                    cesql: 
+                    Issue url: docs: translate cesql/spec.md to zh-CN
+                    Issue url: docs: translate cesql/cesql_tck/README.md to zh-CN
+                    ......
+```
+
+## How to translate
+
+### Step 1. Choose a document to translate 
+
+A translator can find a document to translate by filtering issues based on 
+`{language-code}-translation` label.
+
+Comment the issue to ask reviewers to assign the task to you.
+
+You're expected to finish translation within one week. The task can be assigned to
+others after one week.
+
+### Step 2. submit your translation
```

Comment:
s/submit/Submit

Reply:
Or ignore these comments, and change step 1 to "choose". We just need to be consistent.